design/src/App.scss

```diff
@@ -255,6 +258,28 @@ $ribbon-color: #9a0017;
         }
     }
 
+    .glogg-ribbon {
+        display: none;
```

Comment:
Why does this have a `display: none;`? This will not be accessible to those using screen-readers, for example.

In future projects, consider using "visually hidden css" (remember the search term to [easily find the code snippet](https://css-tricks.com/inclusively-hidden/)) :)